frontend/src/components/PrevNext.js

```diff
@@ -1,25 +1,51 @@
-import React from 'react';
-import Data from './data.json'
-
-
-function PrevNext() {
-  
-    return (
-      <div>
-        <div>
-          {
-            Data.map(post => {
-              return <h4> {post.text}</h4>
-            })
-          }
-        </div>
+import React, { useState, useCallback } from "react";
+import Data from "./data.json";
+import "./styles/BodyStyle.css";
+
+const PrevNext = () => {
+  const [selectedID, setSelectedID] = useState(-1);
+  const toggleSelectedID = useCallback(
+    (index) => {
+      if (selectedID === index) {
+        setSelectedID(-1);
+      } else {
+        setSelectedID(index);
+      }
+    },
+    [selectedID]
+  );
+
+  const questionsAnswers = Data?.Questions?.map((question, index) => {
+    return {
+      question: question.text,
+      answer: Data.Answers[index].text,
+      id: index + 1,
+    };
+  });
```

Comment:
If the data are formatted correctly in the Data.json file as @Sanix-Darker  and @elhmn  suggested here https://github.com/osscameroon/hawaBot/issues/5#issue-953699647

You won't need to reformat the data